src/components/msfvenom/msfvenom.tsx

```diff
@@ -31,66 +34,79 @@ interface FormValuesType {
+        "windows/shellReverseTcp",
         "windows/exec",
         "windows/adduser",
-        "windows/messagebox",
-        "windows/format_all_drives",
-        "windows/loadlibrary",
-        "windows/pingback_reverse_tcp",
-        "windows/speak_pwned",
-        "windows/upexec/bind_tcp",
+        "windows/messageBox",
+        "windows/formatAllDrives",
+        "windows/loadLibrary",
+        "windows/pingBackReverseTcp",
+        "windows/speakPwned",
+        "windows/upexec/bindTcp",
     ],
+    // Options for windows/x64
     [
-        "windows/x64/meterpreter/reverse_tcp",
-        "windows/x64/shell_reverse_tcp",
+        "windows/x64/meterpreter/reverseTcp",
+        "windows/x64/shellReverseTcp",
         "windows/x64/exec",
-        "windows/x64/adduser",
-        "windows/x64/messagebox",
-        "windows/x64/loadlibrary",
-        "windows/x64/peinject/bind_tcp",
-        "windows/x64/shell/bind_named_pipe",
+        "windows/x64/addUser",
+        "windows/x64/messageBox",
+        "windows/x64/loadLibrary",
+        "windows/x64/peinject/bindTcp",
+        "windows/x64/shell/bindNamedPipe",
     ],
+    // Options for linux/x86
     [
-        "linux/x86/meterpreter/reverse_tcp",
-        "linux/x86/shell_bind_tcp",
+        "linux/x86/meterpreter/reverseTcp",
+        "linux/x86/shell_bindTcp",
         "linux/x86/exec",
-        "linux/x86/shell_find_tag",
-        "linux/x86/adduser",
+        "linux/x86/shellFindTag",
+        "linux/x86/addUser",
         "linux/x86/chmod",
-        "linux/x86/read_file",
-        "linux/x86/shell_find_port",
-        "linux/x86/meterpreter/reverse_ipv6_tcp",
+        "linux/x86/readFile",
+        "linux/x86/shellFindPort",
+        "linux/x86/meterpreter/reverseIpv6Tcp",
     ],
+    // Options for Mac/x64
     [
-        "osx/x64/meterpreter/reverse_https",
-        "osx/x64/shell_bind_tcp",
+        "osx/x64/meterpreter/reverseHttps",
+        "osx/x64/shellBindTcp",
         "osx/x64/exec",
         "osx/x64/say",
-        "osx/x64/shell_find_tag",
-        "osx/x64/dupandexecve/bind_tcp",
-        "osx/x64/meterpreter/reverse_tcp",
+        "osx/x64/shellFindTag",
+        "osx/x64/dupandexecve/bindTcp",
+        "osx/x64/meterpreter/reverseTcp",
     ],
+    // Options for Android ARM
     [
-        "android/meterpreter/reverse_https",
-        "android/meterpreter/reverse_tcp",
-        "android/meterpreter_reverse_http",
-        "android/meterpreter_reverse_https",
-        "android/meterpreter_reverse_tcp",
-        "android/shell/reverse_http",
-        "android/shell/reverse_https",
-        "android/shell/reverse_tcp",
+        "android/meterpreter/reverseHttps",
+        "android/meterpreter/reverseTcp",
+        "android/meterpreterReverseHttp",
+        "android/meterpreterReverseHttps",
+        "android/meterpreterReverseTcp",
+        "android/shell/reverseHttp",
+        "android/shell/reverseHttps",
+        "android/shell/reverseTcp",
```

Comment:
All of these changes will need to be undone. These are the arguments passed to `msfvenom` to determine what type of payload should be used.

As an example, if you try to run `msfvenom` with the payload (`-p`) "`windows/meterpreter/reverseTcp`", the command will fail as the payload is invalid. The snake-case used here is what `msfvenom` uses to denote it's payloads, so please change them back.